### PR TITLE
Add WeightRetargeting to Xprize Semifinal yarpmanager Application File

### PR DESCRIPTION
**Pull request by RiccardoGrieco**

### CODE REVIEW

File: app/scripts/Xprize-UserInterfacesRetargeting.xml [hunk 1]
```diff
@@ -123,5 +119,11 @@
     <to>/joypadDevice/Oculus/rpc</to>
     <protocol>fast_tcp</protocol>
   </connection>
+  
+  <connection>
+    <from>/WeightRetargeting/output:o</from>
+    <to>/iFeelSuit/WearableActuatorsCommand/input:i</to>
+    <protocol>udp</protocol>
```
Comment (kouroshD) [inline app/scripts/Xprize-UserInterfacesRetargeting.xml:126]: Depending on where we run the module, at the moment we may not have the udp option for the carrier type. Now, as @S-Dafarra  was mentioning before, we cannot have udp for the double-network configuration. In our case, I think the module will run on robot side, so udp is not a reliable solution. We may use fast_tcp, what do you say @RiccardoGrieco ?

File: app/scripts/Xprize-UserInterfacesRetargeting.xml [hunk 2]
```diff
@@ -97,18 +97,14 @@
...
-    <name>iFeelSuitSkinActuatorsTestApplication</name>
+    <name>WeightRetargetingModule</name>
+    <parameters>--from WeightRetargetingElbows.ini</parameters>
     <environment>YARP_FORWARD_LOG_ENABLE=1</environment>
-    <description>Run iFeelSuit sking application</description>
+    <description>Run weight retargeting application</description>
     <node>localhost</node>
-    <tag>iFeelSuit-SkinActuatorsTestApplication</tag>
-    <dependencies>
-      <port timeout="11.0">/iFeelSuit/WearableActuatorsCommand/input:i</port>
```
Comment (kouroshD) [inline app/scripts/Xprize-UserInterfacesRetargeting.xml:108]: Maybe we can keep the dependencies that are valid for this module, what do you say @RiccardoGrieco ?!

**Review by kouroshD — APPROVED** [review]

**Comment by kouroshD** [comment]
thank you @RiccardoGrieco . I made a small comment, when you are done, let me know, I will merge the PR. thanks.

File: app/scripts/Xprize-UserInterfacesRetargeting.xml (hunk 1 above)
Comment (S-Dafarra) [reply app/scripts/Xprize-UserInterfacesRetargeting.xml:126]: To be honest, I am not sure anymore about this. I remember that the audio did not work when using ``udp``. Maybe it was an isolated case, or maybe it got fixed in the meantime. In any case, I agree that ``fast_tcp`` is safer

**Comment by RiccardoGrieco** [comment]
Thanks @kouroshD for the heads-up!

If you agree I would wait for the following issues to be closed to merge the PR, since for instance the configuration files might change.

- https://github.com/ami-iit/element_weight-retargeting/issues/14
- https://github.com/ami-iit/element_weight-retargeting/issues/17

File: app/scripts/Xprize-UserInterfacesRetargeting.xml (hunk 1 above)
Comment (RiccardoGrieco) [reply app/scripts/Xprize-UserInterfacesRetargeting.xml]: Done in fdd66be

File: app/scripts/Xprize-UserInterfacesRetargeting.xml (hunk 2 above)
Comment (RiccardoGrieco) [reply app/scripts/Xprize-UserInterfacesRetargeting.xml:108]: I agree! Done in c88f246.

**Comment by RiccardoGrieco** [comment]
>@RiccardoGrieco I would say, we can try to have the latest commands that allows us to run those codes. Later, we can easily update the yarpmanager application. If still they are the same, please let's merge this PR. I would like to merge the upstream branch to devel.

@kouroshD changed the name of the config file to a generic one, so we can keep using it while correcting/deciding the internal parameters. Should have used it earlier 😄.

The PR can be merged.

**Pull request merged**